src/QMCWaveFunctions/PlaneWave/PWOrbitalSetT.h

```diff
@@ -0,0 +1,130 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2016 Jeongnim Kim and QMCPACK developers.
+//
+// File developed by: Jeongnim Kim, jeongnim.kim@gmail.com, University of Illinois at Urbana-Champaign
+//                    Jeremy McMinnis, jmcminis@gmail.com, University of Illinois at Urbana-Champaign
+//                    Mark Dewing, markdewing@gmail.com, University of Illinois at Urbana-Champaign
+//
+// File created by: Jeongnim Kim, jeongnim.kim@gmail.com, University of Illinois at Urbana-Champaign
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+/** @file PWOrbitalSetT.h
+ * @brief Definition of member functions of Plane-wave basis set
+ */
+#ifndef QMCPLUSPLUS_PLANEWAVE_ORBITALSET_BLAS_H
+#define QMCPLUSPLUS_PLANEWAVE_ORBITALSET_BLAS_H
+
+#include "QMCWaveFunctions/PlaneWave/PWBasis.h"
+#include "type_traits/complex_help.hpp"
+#include "QMCWaveFunctions/SPOSetT.h"
+#include "CPU/BLAS.hpp"
+
+namespace qmcplusplus
+{
+
+template<class T>
+class PWOrbitalSetT : public SPOSetT<T>
+{
+  
+public:
+  using RealType = QMCTraits::RealType;
+  using ComplexType = QMCTraits::ComplexType;
```

Comment:
Replace `ComplexType` with `T` and change `RealType` to
```suggestion
  using RealType = typename RealAlias_impl<T>::value_type;
```
( you need to `#include "type_traits/complex_help.hpp"`

Reply:
@williamfgc Would you confirm if the above is correct?

Reply:
I have altered the code to reflect this suggestion, still waiting for confirmation by @williamfgc

Reply:
Just reuse `SPOSet<T>::RealType`